hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleInjectionModule.java

```diff
@@ -16,13 +16,7 @@
 
 package com.hedera.node.app.throttle;
 
-import dagger.Binds;
 import dagger.Module;
-import javax.inject.Singleton;
 
 @Module
-public interface ThrottleInjectionModule {
-    @Binds
-    @Singleton
-    ThrottleAccumulator bindThrottleAccumulator(ThrottleAccumulatorImpl throttleAccumulator);
-}
+public interface ThrottleInjectionModule {}
```

Comment:
Should I delete the whole class or leave it for future additions?

Reply:
Delete it imo. 😁